roles/test_operator/tasks/tobiko-tests.yml

```diff
@@ -105,3 +105,52 @@
       data:
         config: "{{ lookup('file', cifmw_openshift_kubeconfig) | b64encode }}"
   when: not cifmw_test_operator_dry_run | bool
+
+- name: Cleanup tobiko workloads
+  block:
+    - name: "cleanup Loadbalancers created by Tobiko tests"
+      shell: |
+        for lb in $({{ cifmw_test_operator_tobiko_openstack_cmd }} loadbalancer list -f value -c 'name' | grep "^tobiko")
```

Comment:
relaying on the name may be problematic if there is more than one resource with the same name. I personally prefer to rely on the uuids so I would do it like:
```
for lb in $({{ cifmw_test_operator_tobiko_openstack_cmd }} loadbalancer list | grep "^tobiko" | awk "{print $2}"); do
....
```